packages/react-hooks/README.md

```diff
@@ -84,3 +84,19 @@ function MockComponent() {
   );
 }
 ```
+
+### `usePrevious()`
+
+This hook will store the previous value of a given variable.
+
+```tsx
+function Score({value}) {
+  const previousValue = usePrevious(value);
+  return (
+    <>
+      <p>{value}</p>
+      {previousValue < value && <p>New record!</p>}
```

Comment:
Nit but I would write the condition the other way around: "If the next scrore is greater than the previous one, we have a new record"

`value > previousValue && <p>We have a new record!</p>`